Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityToolkit.SDK.Input.asmdef

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Microsoft.MixedReality.Toolkit.SDK",
+    "name": "Microsoft.MixedReality.Toolkit.SDK.Input",
```

Comment:
We probably need both a Microsoft.MixedReality.Toolkit.SDK for the base assembly as well as Microsoft.MixedReality.Toolkit.SDK.Input

Reply:
I like this. We should consider putting common scripts in Microsoft.MixedReality.Toolkit.SDK (when we have them in the tree)

Reply:
I can start adding in some Common scripts before we merge this and create the base assembly.

Reply:
Its no worries, having the custom assembly ahead of time shouldn't impact anything.

Reply:
Let's add new scripts in a separate PR

Reply:
Having it there displayed an error in Unity which is why I removed it.

Reply:
Also, I'm not so sure about starting on the SDK layer until we've finished most of the work on the Core feature areas.

Reply:
This is actually intended for beta. Don't have a branch for that work... yet :)